backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/report/DocOutboundConfigRepository.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2024 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.report;
+
+import com.google.common.collect.ImmutableSet;
+import de.metas.util.Check;
+import de.metas.util.Services;
+import de.metas.util.StringUtils;
+import lombok.NonNull;
+import org.adempiere.ad.column.AdColumnId;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.util.proxy.Cached;
+import org.compiere.model.I_AD_Column;
+import org.compiere.model.I_AD_PrintFormat;
+import org.compiere.model.I_C_Doc_Outbound_Config;
+import org.compiere.model.I_C_Doc_Outbound_Config_CC;
+import org.compiere.util.Env;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Properties;
+
+import static de.metas.common.util.CoalesceUtil.coalesce;
+
+@Repository
+public class DocOutboundConfigRepository
+{
+	public static final IQueryBL queryBL = Services.get(IQueryBL.class);
+
+	public I_C_Doc_Outbound_Config getConfigById(final int docOutboundConfigId)
+	{
+		return InterfaceWrapperHelper.load(docOutboundConfigId, I_C_Doc_Outbound_Config.class);
+	}
+
+	// note that this method doesn't directly access the DB. Therefore, a unit test DAO implementation can extend this
```

Comment:
LE: ok, i later saw you moved this from `de.metas.document.archive.api.impl.DocOutboundDAO`
I think think calling it repo is a bit unfortunate..i would expect a repo to read and write domain-objects. but i don't have a better idea right now